.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: python
+
+python:
+  - "3.6"
+
+install:
+  - cd api/v1/
+  - pip install -r requirements.txt
+  - pip install coveralls
+
```

Comment:
Keeping the requirements file at the top level package is a recommended way.
Your collaborators will have an easy time locating it and installing your project requirements. 
You won't need the `cd api/v1` command in the .travis.yml too